docs/firestore/usage/index.md

```diff
@@ -239,6 +239,43 @@ firestore()
 To learn more about all of the querying capabilities Cloud Firestore has to offer, view the
 [Firebase documentation](https://firebase.google.com/docs/firestore/query-data/queries).
 
+It is now possible to use the `Filter` instance to make queries. They can be used with the existing query API.
+For example, you could chain like so:
+
+```js
+const snapshot = await firestore()
+  .collection('Users')
+  .where(Filter('user', '==', 'Tim'))
+  .where('email', '==', 'tim@email.com')
```

Comment:
Going to make a batch of suggestions + commit them that changes these to the RFC-prescribed example domain: https://datatracker.ietf.org/doc/html/rfc2606#section-3

```suggestion
  .where('email', '==', 'tim@example.com')
```